packages/sdk/src/handlers/contract-kit.ts

```diff
@@ -12,10 +12,11 @@ import { ChainHandler } from './interface';
  */
 export class ContractKitTransactionHandler implements ChainHandler {
   private signedTransaction?: EncodedTransaction;
-  private readonly blockchainAddress;
-  private readonly dekAddress;
+  private readonly blockchainAddress: string;
+  private readonly dekAddress: string;
 
   constructor(private readonly kit: ContractKit) {
+    this.kit.contracts.getAccounts()
```

Comment:
side-effect or left-over? maybe remove it?

Reply:
Woops